docs/development.rst

```diff
@@ -105,6 +105,15 @@ GroupServer_. Its usage_ is simple (which is why we use
 ``mr.developer``) and its behaviour is controlled by the
 `development configuration file`_.
 
+Installation
```

Comment:
For newer installs, at least, Mr Developer is installed as part of the build process, so using `pip` is unnecessary.